test/scripts/install_chromedriver_travis.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ev
+
+CHROME_VERSION="google-chrome-stable"
+CHROME_DRIVER_VERSION=2.31
+
+echo "Uninstalling current Chromium from `which chromium-browser`..."
+sudo apt-get purge chromium-browser
+rm ~/.config/chromium/ -rf
+echo "Uninstalling current Google Chrome from `which google-chrome`..."
+sudo apt-get purge google-chrome-stable
+rm ~/.config/google-chrome/ -rf
+
+echo "Installing $CHROME_VERSION..."
+sudo wget --no-verbose -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
```

Comment:
This log doesn't make sense to me - can you explain?

Reply:
This was initially taken from an implementation of Dockerfile in our cucumber repository which was driven for purposes of running tests on headless Chrome that required latest beta version at the time (https://bitbucket.org/onfido/cucumber/src/2adc5cb302a0db505bc8d165e81005e4c5e25d95/Dockerfile?at=master&fileviewer=file-view-default). Since we're currently not using headless Chrome due to an issue on Travis + the browser is now stable, we could delete some of these lines - I'll experiment a little to see if it still works